### PR TITLE
Gke 1.1 features

### PR DESCRIPTION
**Pull request by j3ffml**
Forgot to check this change in, test has been running config from my fork.

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -1347,6 +1360,7 @@ export KUBE_GKE_NETWORK=${E2E_NETWORK}
 export E2E_SET_CLUSTER_API_VERSION=${E2E_SET_CLUSTER_API_VERSION:-}
 export DOGFOOD_GCLOUD=${DOGFOOD_GCLOUD:-}
 export CMD_GROUP=${CMD_GROUP:-}
+export MACHINE_TYPE=${MINION_SIZE:-}
```
Comment (ihmccreery) [inline hack/jenkins/e2e.sh]: Add a comment explaining why?

**Comment by jlowdermilk** [comment]
cc @ixdy

**Comment by ihmccreery** [comment]
LGTM.

File: hack/jenkins/e2e.sh [hunk 2]
```diff
@@ -691,6 +691,19 @@ case ${JOB_NAME} in
...
+    : ${DOGFOOD_GCLOUD:="true"}
+    : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
+    : ${E2E_CLUSTER_NAME:="gke-1-1-features"}
+    : ${E2E_NETWORK:="gke-1-1-features"}
+    : ${E2E_SET_CLUSTER_API_VERSION:=y}
+    : ${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
+    : ${PROJECT:="k8s-jkns-e2e-gke-1-1-features"}
+    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=GCE\sL7\sLoadBalancer\sController|Job|Horizontal\spod\sautoscaling"}
+    MINION_SIZE="n1-standard-2"
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: isn't n1-standard-2 the default?

Comment (jlowdermilk) [inline hack/jenkins/e2e.sh]: Not for GKE. If it isn't specified, it will use server default, which is n1-standard-1.

**Comment by ixdy** [comment]
ok, lgtm

File: hack/jenkins/e2e.sh (hunk 2 above)
Comment (ixdy) [inline hack/jenkins/e2e.sh]: ... though maybe it's worth adding a comment why we need more than the default here? (the other GKE projects don't override as far as I can tell)

Comment (jlowdermilk) [inline hack/jenkins/e2e.sh]: I added a comment that the reason this test needs larger nodes is to have enough cpu/ram to run the HPA tests.

**Pull request merged**